tests/teamcity/run-server.sh

```diff
@@ -20,7 +20,7 @@ source $DIRNAME/$FXA_TEST_NAME
 
 # optionally, GIT_COMMIT can be set in the environment to override
 if [ -z "$GIT_COMMIT" ]; then
-  GIT_COMMIT=$(curl -s "$FXA_CONTENT_ROOT/ver.json" | jsawk  "return this.commit")
+  GIT_COMMIT=$(curl -s "$FXA_CONTENT_VERSION" | jsawk  "return this.commit" | perl -pe 's/OUT:\s+//')
```

Comment:
Which 's@$FXA_CONTENT_ROOT/ver.json@$FXA_CONTENT_VERSION@, or the perl oneliner to strip 'OUT: ' from jsawk output? 

I suppose I don't need to change the former. The latter is explained in the initial PR.